src/channels/pages/ChannelDetailsPage/ChannelDetailsPage.tsx

```diff
@@ -40,6 +38,7 @@ export interface ChannelDetailsPageProps {
   searchShippingZones: (query: string) => void;
   searchShippingZonesData?: SearchData;
   fetchMoreShippingZones: FetchMoreProps;
+  channelShippingZones?: ChannelShippingZones;
```

Comment:
Is it optional or just can be undefined?

Reply:
Is optional, channel details page is used by update and create, and in create we don't have any shipping zones assigned